purchases/src/test/java/com/revenuecat/purchases/attributes/SubscriberAttributesPurchasesTests.kt

```diff
@@ -0,0 +1,138 @@
+package com.revenuecat.purchases.attributes
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.revenuecat.purchases.IdentityManager
+import com.revenuecat.purchases.Purchases
+import io.mockk.Runs
+import io.mockk.every
+import io.mockk.just
+import io.mockk.mockk
+import io.mockk.slot
+import io.mockk.verify
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.util.concurrent.ExecutorService
+
+@RunWith(AndroidJUnit4::class)
+class SubscriberAttributesPurchasesTests {
+    private lateinit var underTest: Purchases
+    private val appUserId = "juan"
+    private val subscriberAttributesManagerMock = mockk<SubscriberAttributesManager>()
+
+    @Before
+    fun setup() {
+        underTest = Purchases(
+            mockk(relaxed = true),
+            appUserId,
+            mockk(relaxed = true),
+            mockk(relaxed = true),
+            mockk(relaxed = true),
```

Comment:
FFTI: named params for these would make them easier to read, although I could see an argument for it not being important given that they're just plain mocks